wiki/en/en-Software-Manual.md

```diff
@@ -261,4 +261,4 @@ your distance to the server is too large or your internet connection is not suff
 Overall Delay is calculated from the current Ping Time and the delay introduced by the current buffer settings.
 
 Audio Upstream Rate depends on the current audio packet size and compression setting. Make sure that the upstream
-rate is not higher than your available internet upload speed (check this with a service such as [speedtest.net](http://speedtest.net)).
+rate is not higher than your available internet upload speed (check this with a service such as [speedtest.net](https://librespeed.org/)).
```

Comment:
Since all changes except this do not need translation every change except this one can be merged into main. 

Other languages do not need to know which typos have been fixed for the English version but they do need to know which links have changed. So librespeedtest will be Part of the next release but your fixes will be part of this release.

Therefore please do not link to librespeedtest yet

Reply:
Reverted it.